Source/WebCore/inspector/agents/InspectorNetworkAgent.h

```diff
@@ -139,7 +139,7 @@ class InspectorNetworkAgent : public InspectorAgentBase, public Inspector::Netwo
     void searchInRequest(Inspector::Protocol::ErrorString&, const Inspector::Protocol::Network::RequestId&, const String& query, bool caseSensitive, bool isRegex, RefPtr<JSON::ArrayOf<Inspector::Protocol::GenericTypes::SearchMatch>>&);
 
 protected:
-    InspectorNetworkAgent(WebAgentContext&, uint32_t);
+    InspectorNetworkAgent(WebAgentContext&, uint32_t, bool);
```

Comment:
this (and the `uint32_t`) should both be given names here since it's not clear what they mean

Reply:
i wonder if we should create a `struct ConfigurationOptions` or something like that instead of passing individually named arguments.  that might be slightly cleaner code to read